PermissionScope/PermissionScope.swift

```diff
@@ -1003,7 +1075,8 @@ typealias resultsForConfigClosure     = ([PermissionResult]) -> Void
         onCancel = cancelled
         
         DispatchQueue.main.async {
-            while self.waitingForBluetooth || self.waitingForMotion { }
+            while
+                self.waitingForBluetooth || self.waitingForMotion { }
```

Comment:
A few questions here:

1. Is this the preferred indentation? (Not necessarily opposed, just wondering why it was changed.)
2. Is this busy-waiting on the main thread? I realize this is existing code, but it still seems strange to me. Should we be concerned about this?